### PR TITLE
Feat/46 create items

### PR DESCRIPTION
**Pull request by Jephuff**

### CODE REVIEW

File: src/components/Content/Content.tsx [hunk 1]
```diff
@@ -54,43 +55,51 @@ class Content extends React.Component<any, any> {
...
+                  onClick={async () => {
+                    await setItem(null, type);
+                    this.setState({ creating: true, editing: false });
+                  }}
                   size="tiny"
-                  style={{ fontWeight: 'normal' }}
+                  style={{ fontWeight: 'bold' }}
                 >
-                  Edit
+                  Create
```
Comment (cheapsteak) [inline src/components/Content/Content.tsx]: is this git diff being dumb?

File: src/components/Groups/GroupsPage.tsx [hunk 2]
```diff
@@ -141,9 +142,9 @@ const render = props => {
             return (
               <Aux>
                 <ListPane
-                  sortableFields={RESOURCE_MAP.users.sortableFields}
+                  sortableFields={RESOURCE_MAP.users.schema.filter(field => field.sortable)}
                   initialSortOrder={RESOURCE_MAP.users.initialSortOrder}
-                  initialSortField={RESOURCE_MAP.users.initialSortField}
+                  initialSortField={RESOURCE_MAP.users.schema.find(field => field.initialSort)}
```
Comment (cheapsteak) [inline src/components/Groups/GroupsPage.tsx]: let's revert this, see my pull https://github.com/kids-first/kf-drp-auth-ui/pull/67

File: src/components/Content/Content.tsx (hunk 1 above)
Comment (Jephuff) [reply src/components/Content/Content.tsx]: Yeah I changed the indentation so thats affecting the diff.

File: src/components/Content/Content.tsx [hunk 3]
```diff
@@ -99,13 +108,15 @@ class Content extends React.Component<any, any> {
...
-                    this.setState({ saving: true });
-                    await saveChanges();
-                    this.setState({ saving: false, editing: false, updates: null });
+                  this.setState({ saving: true });
+                  const newItem = await saveChanges();
+                  if (newItem.id === id) {
+                    this.setState({ ...INITIAL_STATE });
                   } else {
-                    this.setState({ editing: true });
+                    history.push(`/${type}/${newItem.id}`);
```
Comment (cheapsteak) [inline src/components/Content/Content.tsx]: `.replace`?

Comment (cheapsteak) [reply src/components/Content/Content.tsx]: or actually nvm, we can do that in a separate pr

**Pull request merged**